### PR TITLE
test(e2e): formatted text component

### PR DESCRIPTION
**Pull request by AntonOlkhovskyi**
#### Please provide a link to the associated issue.
https://github.com/SAP/fundamental-ngx/issues/4313
#### Please provide a brief summary of this pull request.
e2e tests for the formatted text component
#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md
- [x] tests for the changes that have been done
- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist

Documentation checklist:
- [x] update `README.md`
- [x] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
- [x] Documentation Examples
- [x] Stackblitz works for all examples

### CODE REVIEW

File: e2e/wdio/core/tests/formatted-text.e2e-spec.ts [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+import {
+    browserIsFirefox,
+    getAlertText,
+    getAttributeByName,
+    getBaseURL,
+    getText,
+    refreshPage,
+    waitForElDisplayed
+} from '../../driver/wdio';
+import formattedTextData from '../fixtures/appData/formatted-text-content';
```
Comment (Seamoo13) [inline e2e/wdio/core/tests/formatted-text.e2e-spec.ts:11]: Please add the object destruction we discussed on the call today

File: e2e/wdio/core/pages/formatted-text.po.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import { CoreBaseComponentPo } from './core-base-component.po';
+import { waitForElDisplayed, waitForPresent } from '../../driver/wdio';
+
+export class FormattedTextPo extends CoreBaseComponentPo {
+    private url = '/formatted-text';
+    root = '#page-content';
```
Comment (Seamoo13) [inline e2e/wdio/core/pages/formatted-text.po.ts:6]: you aren't using 'root' anywhere, please delete it

File: e2e/wdio/core/fixtures/appData/formatted-text-content.ts [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    html_input_second: '<a href="https://www.sap.com" target="_blank">Link to sap.com with target "_self" (will change if already exists)</a>',
+    loripsum_link_href: 'http://loripsum.net/',
+    google_link_href: 'google.com',
+    sap_link_href: 'https://www.sap.com',
+    anchor_href: '#target1',
+    target_self: '_self',
+    target_blank: '_blank',
+    custom_style_1: 'color: red;',
+    custom_style_2: 'color: rgb(26, 13, 171); font-size: 14px;',
+    custom_style_2_FF: 'color:#1a0dab;font-size:14px;'
```
Comment (Seamoo13) [inline e2e/wdio/core/fixtures/appData/formatted-text-content.ts:42]: did you check that these custom_styles work on safari? Just thinking ahead, we won't have safari and IE skipped forever.

**Review by Seamoo13 — CHANGES_REQUESTED** [review]

File: e2e/wdio/helper/common-helper.ts [hunk 4]
```diff
@@ -1,3 +1,10 @@
+import { click } from '../driver/wdio';
+
+export function checkLinkTargetDestination(element, site: string): void {
```
Comment (nikvalor) [inline e2e/wdio/helper/common-helper.ts:3]: element: string

**Review by Betrozov — APPROVED** [review]

**Review by Seamoo13 — APPROVED** [review]

File: e2e/wdio/core/tests/formatted-text.e2e-spec.ts [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+
+describe('Formatted text component', function() {
+    const formattedTextPage = new FormattedTextPo();
+    const { redListItem, convertedLinks, inputHtmlText, secondInputHtmlText } = new FormattedTextPo();
+    const {
```
Comment (valorkin) [inline e2e/wdio/core/tests/formatted-text.e2e-spec.ts:16]: this can be done when you import

File: e2e/wdio/core/tests/formatted-text.e2e-spec.ts [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+        } catch (e) {
+            if (browserIsFirefox()) {
+                // FF driver trows incorrect error for missing alert
+                expect(e.message).toContain('unknown error');
+                return;
+            }
+            expect(e.message).toContain('no such alert');
+            return;
+        }
+        expect(true).toBe(false, 'Alert is present on the screen ');
```
Comment (valorkin) [inline e2e/wdio/core/tests/formatted-text.e2e-spec.ts:71]: why what and how?

**Review by valorkin — APPROVED** [review]
LGTM

**Pull request merged**